tests/test_dtcwt.py

```diff
@@ -81,12 +81,13 @@ def test_skip_hps(skip_hps, normalize):
     assert torch.allclose(lp, lp_ap)
 
 
+@pytest.mark.parametrize("J", range(3))
 @pytest.mark.parametrize("alternate_gh", [True, False])
-def test_phi(alternate_gh):
+def test_phi(J, alternate_gh):
     '''
     Test the low-pass output phi doesn't diverge.
     '''
-    tfm = murenn.DTCWT(alternate_gh=alternate_gh, include_scale=True, skip_hps=True)
+    tfm = murenn.DTCWT(J=J+1, alternate_gh=alternate_gh, include_scale=True, skip_hps=True)
```

Comment:
i'd prefer if you had `J=J` in the callsite and `range(1, 4)` in the `pytest .mark` parametrization macro